application/F3DOptionsTools.cxx

```diff
@@ -90,15 +90,16 @@ static inline const std::array<CLIGroup, 8> CLIOptions = {{
       { "grid-subdivisions", "", "Number of grid subdivisions", "<value>", "" },
       { "grid-color", "", "Color of main grid lines", "<R,G,B>", "" },
       { "edges", "e", "Show cell edges", "<bool>", "1" },
+      { "armature", "", "Enable armature visualization", "<bool>", "1" },
       { "camera-index", "", "Select the camera to use", "<index>", "" },
       { "trackball", "k", "Enable trackball interaction", "<bool>", "1" },
       { "invert-zoom", "", "Invert zoom direction with right mouse click", "<bool>", "1" },
       { "animation-autoplay", "", "Automatically start animation", "<bool>", "1" },
       { "animation-index", "", "Select animation to show", "<index>", "" },
       { "animation-speed-factor", "", "Set animation speed factor", "<factor>", "" },
       { "animation-time", "", "Set animation time to load", "<time>", "" },
-      {"font-file", "", "Path to a FreeType compatible font file", "<file_path>", ""},
-      {"command-script", "", "Path to a script file containing commands to execute", "<file_path>", "" } } },
+      { "font-file", "", "Path to a FreeType compatible font file", "<file_path>", ""},
+      { "command-script", "", "Path to a script file containing commands to execute", "<file_path>", "" } } },
```

Comment:
this whole file is incorrect formatted btw :)

Reply:
I know but only those two lines have no space so it triggered me :P

Reply:
In this section, true ^^